infra/modules/azure_cosmos_account/cosmos.tf

```diff
@@ -0,0 +1,46 @@
+resource "azurerm_cosmosdb_account" "this" {
+  name                          = "${local.app_name_prefix}-cosno-${var.environment.instance_number}"
+  location                      = var.environment.location
+  resource_group_name           = var.resource_group_name
+  offer_type                    = "Standard"
```

Comment:
can you add a comment about why we use 'standard'?

Reply:
It's the only available value as far as I know